recipes/zephyr-7b/README.md

```diff
@@ -0,0 +1,29 @@
+
+# Instructions to Replicate Zephyr 7B
+
+As described in the Zephyr [technical report](https://huggingface.co/papers/2310.16944), training this model proceeds in two steps:
+
+1. Apply SFT to fine-tune Mistral 7B on the UltraChat dataset.
```

Comment:
mention that they are filtered versions and link to the dataset repos?

Reply:
Good idea, done in https://github.com/huggingface/alignment-handbook/pull/11/commits/4b0769d13721b7288a0bb0412e0107196a9cd862